crates/rpc/rpc-eth-types/src/error.rs

```diff
@@ -305,8 +305,15 @@ pub enum RpcInvalidTransactionError {
     #[error("max initcode size exceeded")]
     MaxInitCodeSizeExceeded,
     /// Represents the inability to cover max cost + value (account balance too low).
-    #[error("insufficient funds for gas * price + value")]
-    InsufficientFunds,
+    #[error(
+        "insufficient funds for gas * price + value: value or fee {value_or_fee}, balance {balance}"
```

Comment:
this doesn't quite match the sample of the issue
this only includes value of fee not `want`

> have 1444840609176611 want 2652000000000000'